numba_cuda/numba/cuda/tests/cudapy/test_bfloat16.py

```diff
@@ -1,5 +1,112 @@
-from numba import cuda, float32
-from numba.cuda.bf16 import bfloat16
+import unittest
+from importlib.util import find_spec
+import numpy as np
+from ml_dtypes import bfloat16 as mldtypes_bf16
```

Comment:
This makes ml_dtypes a hard requirement for testing. Even if we're not necessarily going to add it as a hard dependency of numba-cuda without some further deliberation / enquiry, maybe it should be added to the test dependencies at least?

Reply:
Yes - we should add this to the test dependencies.